docs/pages/preview/upcoming-releases.mdx

```diff
@@ -40,8 +21,7 @@ The Database Access team adds MongoDB protocol support.
 
 | Version | Date | Description |
 | - | - | - |
-| First alpha | June 1st, 2021 | Good for some testing and demos. |
-| First beta | June 15th | Deploy on staging. |
+| First beta | July 1st, 2021 | Deploy on staging. |
 | Release | July 15th, 2021 | Good to go for production. |
```

Comment:
@klizhentas We pushed this back by two weeks to get https://github.com/gravitational/teleport/issues/2838 in.